### DISCUSSION TITLE
Import morphOne relationship

### DISCUSSION THREAD

**Discussion by sergioraldana** [konnco/filament-import#31] [Q&A] [answered]
Hello I have a Student resource with morphOne relationship to Profile class. I want to import Students but Profile fields are not being created, only Students field are. In Student resource form Fieldset with Profile relasionship is able to creare both Student/Profile tuples

**Comment by frankyso** [comment, marked as answer]
hello @sergioraldana currently its not available, but you can try the mutateAfterCreate functions to do the operation manually

**Reply by frankyso** [reply]
@sergioraldana please try this version https://github.com/konnco/filament-import/releases/tag/1.4.0

i have added the handleRecordCreation functions, so you can put your own code

**Reply by sergioraldana** [reply]
> @sergioraldana please try this version https://github.com/konnco/filament-import/releases/tag/1.4.0
> 
> i have added the handleRecordCreation functions, so you can put your own code

Hi, thanks, Im trying to use handleRecordCreation but I have a problem with $data, when a row is empty the value is missing, I get Key not found
